### PR TITLE
Update skeleton files to symfony 6 compatibility

### PR DESCRIPTION
**Pull request by alexander-schranz**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | fixes #issuenum
| Related issues/PRs | https://github.com/sulu/sulu/pull/6563
| License | MIT
| Documentation PR | sulu/sulu-docs#prnum

#### What's in this PR?

Update skeleton files to symfony 6 compatibility.

#### Why?

Skeleton should always be update with the latest symfony recipe changes.

### CODE REVIEW

**Comment by alexander-schranz** [comment]
PHP 8.0 is failing because the skeleton is build with latest Symfony Recipes, which are only compatible with Symfony 6.1. The question is if we drop PHP 8.0 in skeleton or if we add a CI hack to test against PHP 8.0 or if we downgrade the recipe. /cc @sulu/core-team

**Comment by niklasnatter** [comment]
> PHP 8.0 is failing because the skeleton is build with latest Symfony Recipes, which are only compatible with Symfony 6.1. The question is if we drop PHP 8.0 in skeleton or if we add a CI hack to test against PHP 8.0 or if we downgrade the recipe. /cc @sulu/core-team

Would not any hacks and just drop 8.0 in the CI 🙂

**Comment by chirimoya** [comment]
> PHP 8.0 is failing because the skeleton is build with latest Symfony Recipes, which are only compatible with Symfony 6.1. The question is if we drop PHP 8.0 in skeleton or if we add a CI hack to test against PHP 8.0 or if we downgrade the recipe. /cc @sulu/core-team

Drop 8.0 for the skeleton.

**Review by niklasnatter — APPROVED** [review]

File: config/services.yaml [hunk 1]
```diff
@@ -12,25 +15,11 @@ services:
...
-    # add controllers as services that are specific to a certain sulu context
-    # a sulu context is a different mode in the kernel allowing to have a different set of services
-    # because controllers might rely on services only existing in one of both contexts
-    App\Controller\Admin\:
-        resource: '../src/Controller/Admin'
-        tags: ['controller.service_arguments', {name: 'sulu.context', context: 'admin'}]
-
-    App\Controller\Website\:
-        resource: '../src/Controller/Website'
-        tags: ['controller.service_arguments', {name: 'sulu.context', context: 'website'}]
```
Comment (ghost) [inline config/services.yaml:33]: Actually, I find it quite useful to automatically set the context here. Why has this been removed?

Comment (alexander-schranz) [reply config/services.yaml:33]: @flogado we are more moving away from a multi kernel setup, which make `sulu.context` tag disappear. Still you can structure your controllers as you want. But we are not forcing this as there are also lot of projects which don't have any admin controllers if they just create websites.

Comment (ghost) [reply config/services.yaml:33]: @alexander-schranz Thank you for your quick reply. 👍 

One of our website controllers has a dependency on another service from Sulu that is only available in the website context. Accordingly, we ran into an error here without this configuration.

I think removing the multi-kernel setup is the right thing to do. But for now, it still exists. - Personally I think the configuration has been removed too early. 😄

Comment (alexander-schranz) [reply config/services.yaml:33]: > One of our website controllers has a dependency on another service from Sulu that is only available in the website context. Accordingly, we ran into an error here without this configuration.

That is good to know. Which one is that? Can you create an issue for it?

**Pull request merged**